CONTRIBUTING.md

```diff
@@ -31,62 +50,61 @@ When submitting an issue please include the following information:
-royalty-free, sublicenseable, transferable license under all of Your relevant intellectual property rights
-(including copyright, patent, and any other rights), to use, copy, prepare derivative works of, distribute and
-publicly perform and display the Contributions on any licensing terms, including without limitation:
-(a) open source licenses like the MIT license; and (b) binary, proprietary, or commercial licenses. Except for the
-licenses granted herein, You reserve all right, title, and interest in and to the Contribution.
+- Check that new and updated code follows Newtonsoft.Json-for-Unity's existing
+  code formatting and naming standard at the [Code style][wiki-codestyle] wiki
+  page.
 
-You confirm that you are able to grant us these rights. You represent that You are legally entitled to grant the
-above license. If Your employer has rights to intellectual property that You create, You represent that You have
-received permission to make the Contributions on behalf of that employer, or that Your employer has waived such
-rights for the Contributions.
+- Run Newtonsoft.Json-for-Unity's unit tests to ensure no existing functionality
+  has been affected. See the
+  [Src/Newtonsoft.Json-for-Unity.Tests/README.md][github-tests-docs] for more
+  info.
 
-You represent that the Contributions are Your original works of authorship, and to Your knowledge, no other person
-claims, or has the right to claim, any right in any invention or patent related to the Contributions. You also
-represent that You are not legally obligated, whether by entering into an agreement or otherwise, in any way that
-conflicts with the terms of this license.
+- Write new unit tests to test your changes. All features and fixed bugs must
+  have tests to verify they work
 
-Kalle Jillheden acknowledges that, except as explicitly described in this Agreement, any Contribution which
-you provide is on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED,
-INCLUDING, WITHOUT LIMITATION, ANY WARRANTIES OR CONDITIONS OF TITLE, NON-INFRINGEMENT, MERCHANTABILITY, OR FITNESS
-FOR A PARTICULAR PURPOSE.
+Read [GitHub Help][pullrequesthelp] for more details about creating pull
+requests.
 
 [github]: https://github.com/jilleJr/Newtonsoft.Json-for-Unity
+[github-bug-report]: https://github.com/jilleJr/Newtonsoft.Json-for-Unity/issues/new?assignees=&labels=bug&template=bug_report.md&title=Bug%3A+
+[github-feature-request]: https://github.com/jilleJr/Newtonsoft.Json-for-Unity/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=Suggestion%3A+
+[github-help-request]: https://github.com/jilleJr/Newtonsoft.Json-for-Unity/issues/new?assignees=&labels=&template=not-working-as-expected.md&title=Help%3A+
+[github-tests-docs]: https://github.com/jilleJr/Newtonsoft.Json-for-Unity/blob/master/Src/Newtonsoft.Json-for-Unity.Tests/README.md
+[gitflowguide]: https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4743215)